pkgs/top-level/python-packages.nix

```diff
@@ -346,6 +346,8 @@ in {
 
   clustershell = callPackage ../development/python-modules/clustershell { };
 
+  CNVkit = callPackage ../applications/science/biology/cnvkit { };
```

Comment:
update ref to python-modules/

Reply:
It's not clear to me, CNVkit is the application and cnvlib the underlying python module. My intention was to package CNVkit as an application rather than cnvlib as a module. Happy to move it to python-modules if you think this is more appropriate.

Reply:
@jbedo My understanding is that everything in pkgs/top-level/python-packages.nix should belong to pkgs/development/python-modules. If we chose to ignore the module and keep `CNVkit` in pkgs/applications/ then it should be imported in pkgs/top-level/all-packages.nix

Reply:
I've moved it to python-modules.